projects/web-app/src/views/sitemap.view.vue

```diff
@@ -1,37 +1,54 @@
 <template>
-  <panel-section title="Sitemaps">
-    <div v-if="!sitemaps.length" class="warning-message">
-      <WarningIcon class="icon" />
-      <p>
-        No sitemap found at
-        <external-link :href="sitemapUrl">{{ sitemapUrl }}</external-link>.
-      </p>
-    </div>
-    <code v-else>
-      <pre>{{ sitemaps }}</pre>
-    </code>
-  </panel-section>
-  <panel-section title="Resources">
-    <ul class="resource-list">
-      <li>
-        <external-link href="https://www.sitemaps.org/">
-          Sitemaps.org
-        </external-link>
-      </li>
-      <li>
-        <external-link href="https://support.google.com/webmasters/answer/183668?hl=en">
-          Build and submit a sitemap - Google Support
-        </external-link>
-      </li>
-    </ul>
-  </panel-section>
+  <div>
+    <panel-section title="Sitemaps">
+      <div v-if="!sitemaps.length" class="warning-message">
+        <WarningIcon class="icon" />
+        <p>
+          No sitemap found at
+          <external-link :href="sitemapUrl">{{ sitemapUrl }}</external-link>.
+        </p>
+      </div>
+      <div
+        v-else
+        v-for="sitemap in sitemaps"
+        :key="Object.keys(sitemap)[0]"
+        class="sitemap-view__sitemap"
+      >
+        <p class="sitemap-view__sitemap-url">{{ Object.keys(sitemap)[0] }}</p>
+        <tree-menu
+          v-if="Object.values(sitemap)[0].elements[0].elements"
+          :name="Object.values(sitemap)[0].elements[0].name"
+          :elements="Object.values(sitemap)[0].elements[0].elements"
+        />
+        <div v-else class="sitemap-view__sitemap-error warning-message">
+          <WarningIcon class="icon" />
+          <p>Could not read/parse the sitemap.</p>
+        </div>
+      </div>
+    </panel-section>
+    <panel-section title="Resources">
+      <ul class="resource-list">
+        <li>
+          <external-link href="https://www.sitemaps.org/">
+            Sitemaps.org
+          </external-link>
+        </li>
+        <li>
+          <external-link href="https://support.google.com/webmasters/answer/183668?hl=en">
+            Build and submit a sitemap - Google Support
+          </external-link>
+        </li>
+      </ul>
+    </panel-section>
+  </div>
 </template>
 
 <script>
 import { computed } from 'vue';
-import useHead from '@/composables/use-head';
-import PanelSection from '@shared/components/panel-section';
 import ExternalLink from '@shared/components/external-link';
+import PanelSection from '@shared/components/panel-section';
+import TreeMenu from '@shared/components/tree-menu';
```

Comment:
This is the only line that is changed right?

Reply:
Yup. `import TreeMenu from '@shared/components/tree-menu';` was added, nothing more.